specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/1.2-preview/iotcentral.json

```diff
@@ -0,0 +1,5643 @@
+        "description": "Get the last telemetry value from a module.",
+        "x-ms-examples": {
+          "Get telemetry from module": {
+            "$ref": "./examples/modules_telemetry_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "$ref": "#/parameters/ModuleName"
+          },
+          {
+            "$ref": "#/parameters/TelemetryName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceTelemetry"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/devices/{deviceId}/properties": {
+      "get": {
+        "operationId": "Devices_GetProperties",
+        "summary": "Get device properties",
+        "description": "Get all property values of a device by device ID.",
+        "x-ms-examples": {
+          "Get properties": {
+            "$ref": "./examples/devices_properties_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DeviceId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceProperties"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Devices_ReplaceProperties",
+        "summary": "Replace device properties",
+        "description": "Replace all property values of a device by device ID.",
+        "x-ms-examples": {
+          "Update properties": {
+            "$ref": "./examples/devices_properties_update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/DeviceId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Device properties.",
+            "schema": {
+              "$ref": "#/definitions/DeviceProperties"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceProperties"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
```

Comment:
Operations with a 202 response should specify `x-ms-long-running-operation: true`.